driver/src/contracts/stablex_contract.rs

```diff
@@ -50,7 +50,7 @@ impl StableXContract for StableXContractImpl {
     fn get_current_auction_index(&self) -> Result<U256> {
         self.base
             .contract
-            .query("getCurrentStateIndex", (), None, Options::default(), None)
+            .query("getCurrentBatchId", (), None, Options::default(), None)
```

Comment:
Was this changed recently?

Reply:
Yes, and not adjusted in the services (because in two separate repos we don't notice until someone tries to update the submodule).